### PR TITLE
Use wrapped DB view creations for stations

### LINKED ISSUE #92: Add measurement layer creation for station_layer_creator
Add `"measurement"` layer creation to `station_layer_creator` service.

Convention: If we detect a "prediction" layer we have to create a `"measurement"`.

Precondition: DB function to cerate measurment view: `station_data.create_prediction_view(...)` -> #91

### PR DESCRIPTION
**Pull request by chrismayer**
This uses the DB view creation function `station_data.create_data_views('STATION-CODE', 'POLLUTANT')`, which creates the prediction view and the measurement view for a station-pollutant-combination. The function was introduced with #108.

Also ensures that the aggregated DB views and GS layers are only tried to create if they are not already existing in the GeoServer.

Closes #92

### CODE REVIEW

**Comment by jdk24** [comment]
### tl;dr: 
`station_layer_creator` creates both prediction_... and measurement_... views beautifully. Experienced odd GeoServer behaviour again where the respective layers are not published. Assuming this is a GeoServer issue as of recent, not a station layer problem. With the recent experience, since this has run for you, I'm voting for merging. 

### Long version

After creating DB views successfully, `station_layer_creator` tries to create GS layer and throws error `400 - Bad Request: Trying to create new feature type inside the store, but no attributes were specified` which points as issues between GeoServer <-> PostGIS. 

The `station_data` store was created as far as the UI is concerned, but throws `Schema &apos;http://station_data:fv_stations&apos; does not exist.` when trying to preview any layer, including initial `fv_stations`. Manually publishing layers via GS GUI throws error above. Manually creating store and publishing layers works. 

So far same behaviour for restarting stack, deleting `geoserver_mnt`, etc. but seems similar to how testing #61 went.

**Review by jdk24 — APPROVED** [review]
### tl;dr: 
`station_layer_creator` creates both prediction_... and measurement_... views beautifully. Experienced odd GeoServer behaviour again where the respective layers are not published. Assuming this is a GeoServer issue as of recent, not a station layer problem. With the recent experience, since this has run for you, I'm voting for merging. 

### Long version

After creating DB views successfully, `station_layer_creator` tries to create GS layer and throws error `400 - Bad Request: Trying to create new feature type inside the store, but no attributes were specified` which points at some issue between GeoServer <-> PostGIS. 

The `station_data` store was created as far as the UI is concerned, but throws `Schema &apos;http://station_data:fv_stations&apos; does not exist.` when trying to preview any layer, including initial `fv_stations`. Manually publishing layers via GS GUI throws error above. Manually creating store and publishing layers works. 

So far same behaviour for restarting stack, deleting `geoserver_mnt`, etc. but seems similar to how testing #61 went.

**Comment by chrismayer** [comment]
@JakobMiksch did some investigations on this in another project and maybe has a workaround for this.

**Comment by JakobMiksch** [comment]
I guess I had the same problem. 
See my [issue](https://github.com/meggsimum/geoserver-node-client/issues/31) in `geoserver-node-client` and this [thread on StackOverFlow](https://gis.stackexchange.com/questions/231259/geoserver-gives-schema-does-not-exist-after-creation-using-rest-interface).

Workaround: I open the GeoServer GUI, open the workspace setting and click "Save". Afterwards the PostGIS layer works as expected.

**Comment by chrismayer** [comment]
Thanks @JakobMiksch. Manual workaround seems acceptable since it is "only" affects the workspace, so only one entity needs to fixed manually.

OK, since there is only a manual workaround at the moment and it is not related to this PR I vote for merging this.

**Comment by jdk24** [comment]
Just in case this hinges on me: I'm all for it still, approval still valid.
